src/code42cli/output_formats.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+import click
+from texttable import Texttable
+
+from code42cli.cmds.enums import OutputFormat
+
+
+def output_format(_, __, value):
+    if value is not None:
+        if value == OutputFormat.CSV:
+            return to_csv
+        if value == OutputFormat.RAW:
+            return to_json
+        if value == OutputFormat.ASCII:
+            return to_ascii_table
+        if value == OutputFormat.JSON:
+            return to_formatted_json
+
+
+output_option = click.option(
+    "-f",
+    "--format",
+    type=click.Choice(OutputFormat(), case_sensitive=False),
+    help="The output format of the result.",
+    callback=output_format,
+)
+
+
+def to_csv(output, header):
+    columns = ",".join(header.values())
+
+    lines = []
+    lines.append(columns)
+    for row in output:
+        items = [str(row[key]) for key in header.keys()]
+        line = ",".join(items)
+        lines.append(line)
+    return "\n".join(lines)
+
+
+def to_ascii_table(output, header):
```

Comment:
use `find_format_width` and `format_to_table` in here instead of using `Texttable`. It probably also makes sense to move the `find_format_width` and `format_to_table` functions into this module.

Reply:
done.